frontend/src/v4/routes/viewerGui/components/measurements/measurements.component.tsx

```diff
@@ -16,7 +16,7 @@
  */
 
 import { PureComponent } from 'react';
-
+import { ViewerScrollArea } from '@/v5/ui/v4Adapter/components/viewerScrollArea.styles';
```

Comment:
This isn't being used anymore. Not that it really matters since V4 code is filled with unused components haha